app/js/components/App.jsx

```diff
@@ -9,13 +9,26 @@
  * under the License.
  * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
  */
+
 import React, {Component} from 'react';
 import {Header} from './common/Header';
 import PageComponent from './page/pageComponent';
 import BreadCrumbComponent from './breadCrumb/breadCrumbComponent';
 import CohortTable from '../components/common/table';
 
+/**
+ * The App Component class
```

Comment:
This description does not add any value because you are simply putting the name of the class in words. :)